File: dev/sbt-plugin/src/sbt-test/sbt-plugin/run-all-scaladsl/p/app/controllers/Application.scala
/*
 * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
 */

package controllers

import play.api.mvc._
import views._

class Application(controllerComponents: ControllerComponents) extends AbstractController(controllerComponents) {
  def index = Action {
    Ok(html.index())
  }
}

File: dev/sbt-plugin/src/sbt-test/sbt-plugin/run-play-on-conf/p/app/controllers/Application.java
/*
 * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
 */

package controllers;

import play.*;
import play.mvc.*;

import views.html.*;

public class Application extends Controller {
    public Result index() {
        return ok(index.render());
    }
}
